pages/agent/v3/aws/autoscaling_mac_metal.md.erb

```diff
@@ -0,0 +1,200 @@
+# Auto Scaling mac1.metal instances
+
+Run your builds on AWS `mac1.metal` using our [CloudFormation template](https://github.com/buildkite/elastic-mac-for-aws).
+This template creates an Auto Scaling group, Launch Template, and Host Resource Group
+suitable for maintaining a pool of EC2 `mac1.metal` instances that run the
+Buildkite Agent. These agents can be used to run Buildkite Pipelines and build
+Xcode based software projects for macOS, iOS, iPadOS, tvOS, and watchOS.
+
+As you must prepare and supply your own AMI for this template, macOS support has
+**not** been incorporated into the Elastic CI Stack for AWS.
+
+Using an Auto Scaling group for your instances ensures booting your macOS
+Buildkite Agents is repeatable, and provides automatic instance replacement when
+hardware failures occur.
+
+## Prerequisites
+
+You should have familiarity with:
+
+* AWS VPCs
+* AWS EC2 AMIs
+* macOS GUI
+
+You must also choose an AWS Region with `mac1.metal` instances available. See
+[Amazon EC2 Mac Instances](https://aws.amazon.com/ec2/instance-types/mac/) and
+[Amazon EC2 Dedicated Hosts Pricing](https://aws.amazon.com/ec2/dedicated-hosts/pricing/)
+for details on which regions have `mac1.metal` dedicated hosts.
+
+See also the [Amazon EC2 Mac instances User Guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-mac-instances.html)
+for more details on AWS EC2 Mac instances.
+
+## Choose a VPC Layout
```

Comment:
Might be worth manually numbering these headings like we do over in https://docs-review-pr-1111.herokuapp.com/docs/integrations/github-enterprise